client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTableFooter/GroupedInvestigations/GroupedInvestigations.tsx

```diff
@@ -25,21 +25,26 @@ const GroupedInvestigations: React.FC<Props> = (props: Props) => {
     });
     
     const groupedInvestigation = useMemo(() => {
-        return invetigationsToGroup.find((investigation: InvestigationTableRow) => investigation.groupId);
-    }, [invetigationsToGroup]);
+        return investigationsToGroup.find((investigation: InvestigationTableRow) => investigation.groupId);
+    }, [investigationsToGroup]);
     
     const title = groupedInvestigation ? 'הוספת חקירות לקיבוץ' : 'קיבוץ חקירות'
 
     const investigationsToDisplay: InvestigationTableRow[] = useMemo(() => {
```

Comment:
`const investigationsToDisplay: InvestigationTableRow[] = useMemo(() => {
        if (groupedInvestigation) {
            const groupedInvestigations = allGroupedInvestigations.get(groupedInvestigation.groupId);
            const investigationsWithoutGroupIds = invetigationsToGroup.filter((investigation: InvestigationTableRow) => !investigation.groupId);
            const investigationsToDisplayUpdateMode = groupedInvestigations && groupedInvestigations.concat(investigationsWithoutGroupIds);
            return investigationsToDisplayUpdateMode ? investigationsToDisplayUpdateMode: [];
        } else {
            return invetigationsToGroup;
        }
    }, [groupedInvestigation, invetigationsToGroup, allGroupedInvestigations])`

Reply:
try this.

Reply:
fixed [here](https://github.com/CoronaTeamMOD/CoronaInvestigation/pull/1189/commits/1b0e1767764ef232c96bf9fcb671393642efffe4)